server/controllers/finance/cost_center.js

```diff
@@ -276,6 +276,49 @@ function del(req, res, next) {
     .done();
 }
 
+// This function searches for account membership at a cost center,
+// and returns an object table that will allow when creating a transaction
+async function getAllCostCenterAccounts() {
+  const accountConfigCostCenter = [];
+
+  const sql = `
+    SELECT cc.id AS cost_center_id, cc.label AS cost_center_label, cc.is_principal, rfc.is_cost, a.id AS account_id,
+      a.label AS accountLabel, a.number As accountNumber, ritem.is_exception, a.type_id
+    FROM cost_center AS cc
+    JOIN reference_cost_center AS rfc ON rfc.cost_center_id = cc.id
+    JOIN account_reference AS ar ON ar.id = rfc.account_reference_id
+    JOIN account_reference_item AS ritem ON ritem.account_reference_id = ar.id
+    JOIN account AS a ON a.id = ritem.account_id;
+  `;
+
+  const sqlGetAccountNotTitle = `
+    SELECT a.id AS account_id, a.label, a.number
+    FROM account AS a
+    WHERE a.type_id <> 6;
```

Comment:
It would be clearer if a constant for '6' was defined and used with an informative variable name.